docs/source/user_guide/tutorials/getting_started.rst

```diff
@@ -638,6 +638,83 @@ added, as specified in the YAML::
    29999  9999   e       11               7               8               9               10               11
    [30000 rows x 8 columns]
 
+Loop Steps
+==========
+EasyLink supports a number of `operators` which enable flexibility in selecting and configuring
+implementations for various record linkage pipeline steps. In this section we will discuss the 
+``LoopStep`` operator, but other operators include ``ParallelStep``, ``ChoiceStep``, and 
+``HierarchicalStep``. A ``LoopStep`` allows the user to loop a step multiple times such that 
+each iteration depends on the previous. An implementation is specified for each iteration -- each iteration 
+can have different implementations, or all can have the same one.
+
+.. note::
+   So that you may have an idea of the functionalities
+   enabled by operators, note that the other operators mentioned allow
+   the user to create pipeline sections with parallelized processing, one of several branching paths, and 
+   substeps respectively.
+
+.. todo::
+   This page references the ``LoopStep`` class in the EasyLink code rather than the operators described in 
+   the Pipeline Schema doc page eg "loop-able sections".
```

Comment:
What is actually to-do from this?  Change this page to reference the operators described in the pipeline schema doc page once it is in the repo?